R/modeltests.R

```diff
@@ -268,13 +268,13 @@ evaluateRuns <- function(model, mydir, gitPath, compScen, email, mattermostToken
     }
     write(sub("\n$", "", printOutput(grsi, lenCols = lenCols, colSep = colSep)), myfile, append = TRUE)
     if (model == "REMIND") {
-      if (grsi[, "RunType"] != "Calib_nash" && grsi[, "Conv"] %in% c("converged", "converged (had INFES)") && ! grepl("1Regi|testOneRegi", i)) {
+      if (! grepl("Calib_nash|1Regi|testOneRegi", grsi[, "RunType"]) && ! grsi[, "Conv"] %in% c("converged", "converged (had INFES)")) {
```

Comment:
What is the `1Regi` pattern for?

Reply:
For `rs2`, we replaced `testOneRegi EUR` with `1Regi` such that it fits into the column while printing in [colRunType()](https://github.com/pik-piam/modelstats/commit/f66b4a616c8115960ef5950185484d4fd40fd969#diff-4e326773087224797d811812e56edd285ea06c2c02660def0de45f29f0c92656), but two month ago we moved that to the [`loopRuns()`](https://github.com/pik-piam/modelstats/blob/master/R/loopRuns.R#L78) function such that the data received contains the full word, only the `rs2` print doesn't. So, yes, we can remove it now.